src/main/java/org/skriptlang/skript/bukkit/entity/mannequin/elements/ExprMannequinBody.java

```diff
@@ -0,0 +1,115 @@
+package org.skriptlang.skript.bukkit.entity.mannequin.elements;
+
+import ch.njol.skript.bukkitutil.NamespacedUtils;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Example;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.util.ValidationResult;
+import ch.njol.util.coll.CollectionUtils;
+import io.papermc.paper.datacomponent.item.ResolvableProfile;
+import io.papermc.paper.datacomponent.item.ResolvableProfile.SkinPatch;
+import net.kyori.adventure.key.Key;
+import org.bukkit.NamespacedKey;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Mannequin;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.entity.mannequin.ResolvableProfileBuilder;
+import org.skriptlang.skript.bukkit.entity.mannequin.SkinPatchBuilder;
+import org.skriptlang.skript.registration.SyntaxRegistry;
+
+@Name("Mannequin Body")
+@Description("""
+	The body texture displayed on a mannequin.
+	The body key is represented as a namespaced key.
+	A namespaced key can be formatted as 'namespace:id' or 'id'. \
+	It can only contain one ':' to separate the namespace and the id. \
+	Only alphanumeric characters, periods, underscores, and dashes can be used.
+	""")
+@Example("set the mannequin body of {_mannequin} to \"custom:body\"")
```

Comment:
are there any vanilla ones we can use for examples so users can see the example work on their server?